### PR TITLE
Jinghan/add field `Values` to API ChannelJoin

### PR DESCRIPTION
**Pull request by jinghancc**
This PR refactors oomstore API `ChannelJoin`:
- add field `Values` to `types.EntityRow`
- refactor offline db method `Join`

TODO:
- refactor oomstore API `Join`
- refactor oomcli command `join`

related to #533

### CODE REVIEW

**Review by lianxmfor — APPROVED** [review]
lgtm

File: internal/database/offline/test_impl/join.go [hunk 1]
```diff
@@ -164,60 +174,103 @@ func prepareRevisionRanges(oneGroup bool) map[string][]*metadata.RevisionRange {
...
+		if isEmpty {
+			return
+		}
+		for _, row := range rows {
+			if !withValue {
+				row.Values = nil
+			}
+			entityRows <- row
 		}
+		//entityRows <- types.EntityRow{
```
Comment (YikSanChan) [inline internal/database/offline/test_impl/join.go:212]: del

**Review by YikSanChan — APPROVED** [review]
lgtm

File: internal/database/offline/test_impl/join.go (hunk 1 above)
Comment (jinghancc) [reply internal/database/offline/test_impl/join.go]: done

**Review by wfxr — APPROVED** [review]
LGTM

**Pull request merged**